src/core/lib/transition-manager.js

```diff
@@ -20,7 +17,7 @@ export const TRANSITION_EVENTS = {
 const DEFAULT_PROPS = {
   transitionDuration: 0,
   transitionEasing: t => t,
-  transitionInterpolator: viewportLinearInterpolator,
+  transitionInterpolator: new ViewportLinearInterpolator(),
   transitionInterruption: TRANSITION_EVENTS.BREAK,
   transitionProps: VIEWPORT_TRANSITION_PROPS,
```

Comment:
I thought the plan was to drop `transitionProps`

Reply:
Yes... I will remove this line